<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barreMenuFiche</class>
 <widget class="QWidget" name="barreMenuFiche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_1">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>71</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/app/valider96</normaloff>:/app/valider96</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>71</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/app/annuler96</normaloff>:/app/annuler96</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>71</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/app/quitter96</normaloff>:/app/quitter96</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>88</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_fenetre">
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LABEL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="image_fenetre">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>79</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="pixmap">
      <pixmap resource="ressources.qrc">:/app/client96</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
